Include/httpClient/async.h

```diff
@@ -64,6 +57,11 @@ typedef struct AsyncBlock
     /// </summary>
     AsyncCompletionRoutine* callback;
 
+    /// <summary>
+    /// Internal use only
+    /// </summary>
+    HANDLE internalHandle;
```

Comment:
Not necessary.  We create one internally and can just use that.

Reply:
Agreed

---
In reply to: [193618417](https://github.com/Microsoft/libHttpClient/pull/186#discussion_r193618417) [](ancestors = 193618417)